### PR TITLE
[MIG][10.0] queue_job garbage collector

### PR DESCRIPTION
**Pull request by thomaspaulb**
Backport of https://github.com/OCA/queue/pull/223

### CODE REVIEW

File: queue_job/data/queue_data.xml [hunk 1]
```diff
@@ -2,6 +2,16 @@
...
     <data noupdate="1">
 
+        <record id="ir_cron_queue_job_garbage_collector" model="ir.cron">
+            <field name="name">Jobs Garbage Collector</field>
+            <field name="interval_number">5</field>
+            <field name="interval_type">minutes</field>
+            <field name="numbercall">-1</field>
+            <field eval="'queue.job'" name="model"/>
+            <field eval="'requeue_stuck_jobs'" name="function"/>
+            <field eval="'()'" name="args"/>
```
Comment (StefanRijnhart) [inline queue_job/data/queue_data.xml:12]: These three evals are not necessary. You can just put the strings as the content of the tags.

**Review by StefanRijnhart — APPROVED** [review]
Just a minor, non blocking comment. Thanks!

**Review by guewen — APPROVED** [review]
Thanks! Could you squash the fixup before we merge please?

**Comment by thomaspaulb** [comment]
@guewen Squashed.

**Pull request merged**